tests/test_api.rs

```diff
@@ -2229,7 +2229,7 @@ fn object_template_set_named_property_handler() {
       assert!(value.is_int32());
       assert!(this.set_internal_field(0, value.into()));
 
-      rv.set_undefined();
```

Comment:
What's the reason for changing these?

Reply:
v8 is DCHECKing that they are booleans now. it always assumed they were booleans, but now it has a DCHECK.

Reply:
Sweet, thanks!